app/views/users/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<p id="notice"><%= notice %></p>
```

Comment:
it will be better to just give pop up message if the notice is only for deletion

Reply:
Ok. Changed.

Added a simple pop up message for notice and alert. However, there seems to be a problem where the footer did not load until the end since the script is not at the end of the html. If you have a better suggestion on how to display the pop up message, please do let me know.

Reply:
hmm then can just put the script at the end of the html? i.e. `app/views/layouts/application.html.erb` ? because there are chances that other view might also need to display alert and/or notice.

Reply:
Ok. Moved.

Currently, instead of notice or alert, the flash message is renamed into popup_alert, so that the script in application.html.erb will only pop up the intended message.